### PR TITLE
docs / win source installation

### PR DESCRIPTION
**Pull request by PtrckM**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- update source build installation for windows
- update typo link
- update filename output
- update support info

image
image
image
image

### CODE REVIEW

**Comment by PtrckM** [comment]
image

image

File: documentation/docs/installation/source/windows.md [hunk 1]
```diff
@@ -3,9 +3,7 @@
 !!! info "Recommended for Developers Only"
     The [downloadable installer](/installation/download/windows/) is the easiest way to run Hummingbot. Installing from source is only recommended for developers who want to access and modify the software code.
 
-Coming soon.
-
-<!-- The Hummingbot code base is designed and optimized for UNIX-based systems such as macOS and Linux. We recommend that Windows users either:
+The Hummingbot code base is designed and optimized for UNIX-based systems such as macOS and Linux. We recommend that Windows users either:
 
 * Install the [Docker version](/installation/docker/windows). Note that it is recommended to use the Docker Toolbox over native Docker in most cases.
 * Install in [the cloud](/installation/cloud) by using a native Linux virtual machine.
```
Comment (dennisocana) [inline documentation/docs/installation/source/windows.md:9]: Remove this line

File: documentation/docs/installation/source/windows.md [hunk 2]
```diff
@@ -3,9 +3,7 @@
 !!! info "Recommended for Developers Only"
     The [downloadable installer](/installation/download/windows/) is the easiest way to run Hummingbot. Installing from source is only recommended for developers who want to access and modify the software code.
 
-Coming soon.
-
-<!-- The Hummingbot code base is designed and optimized for UNIX-based systems such as macOS and Linux. We recommend that Windows users either:
+The Hummingbot code base is designed and optimized for UNIX-based systems such as macOS and Linux. We recommend that Windows users either:
```
Comment (dennisocana) [inline documentation/docs/installation/source/windows.md:7]: * Follow [Windows Binary Installation Guide](https://docs.hummingbot.io/installation/download/windows/) to download and install using setup.exe installer binary.

File: documentation/docs/installation/source/windows.md [hunk 3]
```diff
@@ -3,9 +3,7 @@
...
```
Comment (dennisocana) [inline documentation/docs/installation/source/windows.md:6]: Installing from source is only recommended for developers who want to access and modify the software code. We recommend that Windows users either:

File: documentation/docs/installation/source/windows.md [hunk 4]
```diff
@@ -3,9 +3,7 @@
 !!! info "Recommended for Developers Only"
     The [downloadable installer](/installation/download/windows/) is the easiest way to run Hummingbot. Installing from source is only recommended for developers who want to access and modify the software code.
```
Comment (dennisocana) [inline documentation/docs/installation/source/windows.md:4]: Remove this note

**Pull request merged**